strawberry/http/async_base_view.py

```diff
@@ -349,6 +352,58 @@ async def run(
                     "Content-Type": "multipart/mixed;boundary=graphql;subscriptionSpec=1.0,application/json",
                 },
             )
+        if isinstance(result, GraphQLIncrementalExecutionResults):
+
+            async def stream() -> AsyncGenerator[str, None]:
+                yield "---"
+
+                response = await self.process_result(request, result.initial_result)
+
+                response["hasNext"] = result.initial_result.has_next
+                response["pending"] = [
+                    p.formatted for p in result.initial_result.pending
+                ]
+                response["extensions"] = result.initial_result.extensions
+
+                yield self.encode_multipart_data(response, "-")
+
+                async for value in result.subsequent_results:
+                    response = {
+                        "hasNext": value.has_next,
+                        "extensions": value.extensions,
+                    }
+
+                    if value.pending:
+                        response["pending"] = [p.formatted for p in value.pending]
+
+                    if value.completed:
+                        response["completed"] = [p.formatted for p in value.completed]
+
+                    if value.incremental:
+                        response["incremental"] = [
+                            {
+                                **p.formatted,
+                                # for Apollo
+                                # content type is `multipart/mixed;deferSpec=20220824,application/json`
+                                "path": ["blogPost"],
+                                "label": "relayTestsBlogPostQuery$defer$relayTestsCommentsFragment",
```

Comment:
this should not be hard-coded, in any case having both label and path seems to be ok for both relay and apollo, so we might want to keep them

tbd if we want to keep the .formatted instead of using our own wrappers

apparently there's no good way to define what spec a client is supporting (Apollo sends `deferSpec=20220824`, but for relay it's up to whoever implements the fetch)